src/main/java/io/codemc/bot/commands/CmdRemove.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2021 CodeMC.io
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+ * documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software,
+ * and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all copies or substantial
+ * portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+ * IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+ * WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE
+ * OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package io.codemc.bot.commands;
+
+import com.jagrosh.jdautilities.command.SlashCommandEvent;
+import io.codemc.api.jenkins.JenkinsAPI;
+import io.codemc.api.nexus.NexusAPI;
+import io.codemc.bot.CodeMCBot;
+import io.codemc.bot.JavaContinuation;
+import io.codemc.bot.utils.CommandUtil;
+import net.dv8tion.jda.api.entities.Guild;
+import net.dv8tion.jda.api.entities.Member;
+import net.dv8tion.jda.api.entities.Role;
+import net.dv8tion.jda.api.exceptions.ErrorHandler;
+import net.dv8tion.jda.api.interactions.InteractionHook;
+import net.dv8tion.jda.api.interactions.commands.OptionMapping;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.OptionData;
+import net.dv8tion.jda.api.requests.ErrorResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+public class CmdRemove extends BotCommand{
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(CmdRemove.class);
+
+    public CmdRemove(CodeMCBot bot){
+        super(bot);
+
+        this.name = "remove";
+        this.help = "Revoke a user's Author status.";
+
+        this.allowedRoles = bot.getConfigHandler().getLongList("allowed_roles", "commands", "service");

```

Comment:
Please name the option properly to match the actual command name.
```suggestion
        this.allowedRoles = bot.getConfigHandler().getLongList("allowed_roles", "commands", "remove");

```

Reply:
I named it `service` because I wanted it to be used as the permission for #8. If you think that should be `accept` instead, then I'll change it.

Reply:
The overall structure mimics the commands themself, so that it is easier for a new user to understand it, if they ever want to use the bot themself.

Perhaps, to prepare for #8 could we make the remove command be a sub-command of `/service` or something similar, just like application is a main command with sub-commands.
You could then have the `service` config option while having remove and alike...

Tho, maybe there's a better aproach than /service for the command then? Like /jenkins? Would fit the overall purpose of the command (Jenkins-related stuff)... Tho maybe not the nexus... Would apreciate your input on this.

Reply:
I wouldn't name it `/jenkins` since it would manage the Nexus state.

I'm assuming subcommands can have differing permissions, so adding it to a `/service` root command could be nice.

Perhaps a `/jenkins` or `/nexus` command could suffice for #9? My main goal would be for users to anonymously read jenkins/nexus state from Discord. I'm unsure how authentication would work for write access, if desired.